packages/lib-classifier/src/plugins/drawingTools/experimental/models/tools/TranscriptionLineTool/TranscriptionLineTool.js

```diff
@@ -7,12 +7,13 @@ const TranscriptionLineTool = types.model('TranscriptionLine', {
   type: types.literal('transcriptionLine')
 })
   .actions(self => {
-    function handlePointerDown (mark) {
-      const allMarks = Array.from(self.marks.values())
-      const mostRecent = allMarks[allMarks.length - 1]
-      mostRecent.initialDrag(mark)
-      mostRecent.finish()
-      return mostRecent
+    function handlePointerDown (event, mark) {
+      mark.initialDrag(event)
+      mark.finish()
+    }
+
+    function handlePointerUp (event, mark) {
+      return
```

Comment:
This works but could maybe check if the mark has a length, and finish if so. We only really want to ignore pointerup if the transcription line mark consists of single point.